keyboards/pica40/keymaps/default/config.h

```diff
@@ -0,0 +1,4 @@
+// Copyright 2022 zzeneg (@zzeneg)
```

Comment:
This file isn't being used and can be removed.

Reply:
Removed.